spec/Dock/Docker/Dns/DnsDockResolverSpec.php

```diff
@@ -32,4 +32,10 @@ function it_removes_the_slash_in_image_name()
     {
         $this->getDnsByContainerNameAndImage('container', 'someone/image:latest')->shouldContain('container.image.docker');
     }
+
+    function it_removes_more_than_one_slash_in_image_name()
+    {
+        $this->getDnsByContainerNameAndImage('container', 'someone/folder/image:latest')
+            ->shouldContain('container.folder_image.docker');
```

Comment:
I do think it should container `container.image.docker` and `image.docker`, nothing related with the folder here :)